documentation/source/users/arkane/installation.rst

```diff
@@ -14,3 +14,27 @@ Note that you'll need to choose between the Basic User or Developer installation
 that are specific to your operating system. Modifying Arkane source code will
 require Developer installation. If you are only looking to run the code, the
 Basic User installation will work.
+
+Installing Q2DTor
+=================
+
+Q2DTor is a software for calculating the partition functions and themodynamic properties of molecular systems with two or more
+torsional modes developed by David Ferro Costas (david.ferro@usc.es) and Antonio Fernandez Ramos (qf.ramos@usc.es) at
+the Universidade de Santiago de Compostela. Arkane can integrate Q2DTor to compute the quantum mechanical partition function 
+of 2D rotors.  
+
+For use of Q2DTor and HinderedRotor2D within Arkane please cite:  
+D. Ferro-Costas, M. N. D. S. Cordeiro, D. G. Truhlar, A. Fernández-Ramos, Comput. Phys. Commun. 232, 190-205, 2018.
+
+A slightly modified version of Q2DTor compatible with Arkane can be installed:  
+
+create a depository for Q2DTor:  
```

Comment:
Please use the format:
```
directives::

    code line 1
    code line 2
```
(here and below)

Reply:
done